crates/task-impls/src/consensus/helpers.rs

```diff
@@ -686,6 +686,7 @@ pub(crate) async fn fetch_proposal<TYPES: NodeType>(
     quorum_membership: Arc<TYPES::Membership>,
     consensus: Arc<RwLock<Consensus<TYPES>>>,
 ) -> Result<Leaf<TYPES>> {
+    tracing::info!("Fetchign proposal for view {:?}", view);
```

Comment:
nit: maybe we should consider logs like these being debug

Reply:
My thinking was that this should be a relatively rare occurrence.  We should only get this if we miss a proposal and are catching back up but I'm not 100%